cmd/lncli/walletrpc_active.go

```diff
@@ -561,6 +561,69 @@ func publishTransaction(ctx *cli.Context) error {
 	return nil
 }
 
+var removeTxCommand = cli.Command{
+	Name: "removetx",
+	Usage: "Attempts to remove the hex encoded transaction from the " +
+		"underlying default wallet.",
+	ArgsUsage: "tx_hex",
+	Description: `
+	Removes the hex-encode raw transaction from the underlying wallet which
+	must still be unconfirmmed (in mempool). This command is useful when a
+	transaction is RBFed by another transaction. The wallet will only 
+	resolve this conflict when the other transaction is mined 
+	(which can take time). If a transaction was removed erronously a simple
+	rebroadcast of the former transaction with the "publishtx" cmd will
+	register the relevant outputs of the raw tx again with the wallet
+	(if there are no errors broadcasting this transaction due to RBF
+	replacement sitting in the mempool). As soon as a removed transaction
+	is confirmed funds will be registered with the wallet again.
+	`,
+	Flags:  []cli.Flag{},
+	Action: actionDecorator(removeTransaction),
+}
+
+func removeTransaction(ctx *cli.Context) error {
+	ctxc := getContext()
+
+	// Display the command's help message if we do not have the expected
+	// number of arguments/flags.
+	if ctx.NArg() != 1 {
+		return cli.ShowCommandHelp(ctx, "removetx")
+	}
+
+	walletClient, cleanUp := getWalletClient(ctx)
+	defer cleanUp()
+
+	tx, err := hex.DecodeString(ctx.Args().First())
+	if err != nil {
+		return err
+	}
+
+	// Deserialize the transaction to get the transaction hash.
+	msgTx := &wire.MsgTx{}
+	txReader := bytes.NewReader(tx)
```

Comment:
See my comment in the previous commit about just requiring the TXID to be specified.